deployment/sklearn/sklearn-census-end-to-end.ipynb

```diff
@@ -23,6 +23,17 @@
    "execution_count": 1,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "# restart your notebook if prompted on Colab\n",
+    "!python -m pip install verta\n",
+    "!python -m pip install wget"
```

Comment:
Put the install steps at the top so we can immediately know whether to restart the notebook if necessary.